src/databricks/labs/ucx/source_code/linters/python_ast.py

```diff
@@ -170,6 +189,51 @@ def do_infer_values(self):
         return self._infer_values()
 
 
+class _ContextualCall(NodeNG):
```

Comment:
```suggestion
class _GetWidgetValue(NodeNG):
```
make name more explicit

Reply:
done